pkg/controller/pravegacluster/pravegacluster_controller.go

```diff
@@ -281,9 +288,12 @@ func (r *ReconcilePravegaCluster) syncSegmentStoreSize(p *pravegav1alpha1.Praveg
 			return fmt.Errorf("failed to update size of stateful-set (%s): %v", sts.Name, err)
 		}
 
-		err = r.syncStatefulSetPvc(sts)
-		if err != nil {
-			return fmt.Errorf("failed to sync pvcs of stateful-set (%s): %v", sts.Name, err)
+		/*We skip calling syncStatefulSetPvc() during upgrade/rollback from version 07*/
+		if !util.IsClusterUpgradingTo07(p) && !r.isRollbackTriggered(p) {
```

Comment:
Same as above. Why can't we use "IsClusterRollbackingFrom07()"?

Reply:
I have changed it